File: sentimental-mario-less/mario.py
from cs50 import get_int

# get user input
while True:
    height = get_int("Enter a positive integer: ")
    if height > 0 and height < 9:
        break

i = 0
j = 0
k = 0
tempheight = height - 1

while i < height:
    k = 0
    # controls spaces
    while k < tempheight:
        print(" ", end="")
        k = k + 1
    # controls hashes
    while j < i + 1:
        print("#", end="")
        j = j + 1
    print()
    i = i + 1
    j = 0
    tempheight = tempheight - 1

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void)
{
    //get input from user
    int height;
    do
    {
        height = get_int("Enter a Positive Number: ");
    }
    while (height < 1 || height > 8);

    //controls new line
    for (int i = height - 1; i >= 0; i--)
    {
        //controls spaces
        for (int j = 0; j < i; j++)
        {
            printf(" ");
        }

        //controls hashes
        for (int k = height; k > i; k--)
        {
            printf("#");
        }

        printf("\n");
    }

}